cmd/crates/soroban-test/tests/it/integration/tx/operations.rs

```diff
@@ -267,19 +268,23 @@ async fn account_merge_with_alias() {
 #[tokio::test]
 async fn set_trustline_flags() {
     let sandbox = &TestEnv::new();
-    let (test, issuer) = setup_accounts(sandbox);
-    let asset = format!("usdc:{issuer}");
-    issue_asset(sandbox, &test, &asset, 100_000, 100).await;
+    let (test, test1) = setup_accounts(sandbox);
+    let asset = "usdc:test1";
```

Comment:
```suggestion
    let asset = format!("usdc:{test1}");
```

just for consistency with line 283, and in case we ever change the name of the issuer in `setup_accounts`

Reply:
`test1` here is the full public key. So this will not test the new feature. But if you want we could update `setup_accounts` to allow naming the local account so that this test isn't dependent on `test1` added implicitly.

Reply:
Oh! I see what you're saying. Yeah, I didn't see that `setup_accounts` adds the accounts with a name "test1".

Maybe we can even just update the `test1` var to `test1_address` or something so that it's clear its the full address vs the name. 

Definitely not a big deal, but I can add an updated suggestion.

Reply:
Sound to me!